### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
To match with what is here: https://github.com/mattermost/mattermost-redux/blob/master/package.json

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -491,6 +318,19 @@ SOFTWARE.
...
+## redux-persist-node-storage
+
+This product contains 'redux-persist-node-storage' by Pelle Jacobs.
+
+Storage adaptor to use node-localstorage with redux-persist
+
+* HOMEPAGE:
+  * https://github.com/pellejacobs/redux-persist-node-storage
+
+* LICENSE:
```
Comment (amyblais) [inline NOTICE.txt:330]: No license

**Comment by amyblais** [comment]
@grundleborg Kind reminder to review (+merge and cherry-pick if approved)

File: NOTICE.txt (hunk 1 above)
Comment (grundleborg) [reply NOTICE.txt:330]: This is the one I meant to comment on previously. NPM says MIT for the license, but given there's no license shown in the Github repo I'm not sure what we can assume.

Comment (amyblais) [reply NOTICE.txt:330]: I've requested that they add a license file: https://github.com/pellejacobs/redux-persist-node-storage/issues/7.

**Pull request merged**